packages/create-block-interactive-template/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+const { join } = require( 'path' );
+
+module.exports = {
+	defaultValues: {
+		slug: 'example-interactive',
+		title: 'Example Interactive',
+		description: 'An interactive block with the Interactivity API',
+		dashicon: 'media-interactive',
+		npmDependencies: [ '@wordpress/interactivity' ],
+		supports: {
+			interactivity: true,
+		},
+		render: 'file:./render.php',
+		viewScript: 'file:./view.js',
+	},
+	variants: {
+		basic: {},
```

Comment:
If there is only a single variant, then w can skip this section and remove all checks like `{{#isBasicVariant}}`.